src/openrct2/ride/gentle/meta/Maze.h

```diff
@@ -63,3 +63,5 @@ constexpr const RideTypeDescriptor MazeRTD =
     SET_FIELD(ConstructionWindowContext, RideConstructionWindowContext::Maze),
 };
 // clang-format on
+
+money64 MazeCalculateCost(money32 constructionCost, const Ride& ride, const CoordsXYZ& _loc);
```

Comment:
The meta header isn't really the best location for this. I would just create a new header for the maze if it doesn't already exist.

Reply:
Moved to a new header file.